src/amchart/Pie.js

```diff
@@ -72,10 +71,11 @@
         this._chart.labelFunction = function(d) {
             return d.title;
         };
-        if (this.marginRight()) { this._chart.marginRight = this.marginRight(); }
-        if (this.marginLeft()) { this._chart.marginLeft = this.marginLeft(); }
-        if (this.marginTop()) { this._chart.marginTop = this.marginTop(); }
-        if (this.marginBottom()) { this._chart.marginBottom = this.marginBottom(); }
+
```

Comment:
@GordonSmith is this ok to do this way? cause margin can be 0 which can relate to false or should i do if this.marginRight() !== false (but idk how the publish param handles a null/blank etc or i figured i would test for all